### PR TITLE
feat: log slash command

### PR DESCRIPTION
**Pull request by nguyennh4522**
**What does this PR do?**

-   [x] feat: log slash command

**Media (Loom or gif)**

https://user-images.githubusercontent.com/104887632/188622368-9cd26ec2-a2f2-4565-bb32-c13193055ebf.mp4

### CODE REVIEW

File: src/commands/config/log_slash/index.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      .setName("log")
+      .setDescription("Monitor guild members' activities")
+
+    data
+      .addSubcommand(info)
+      .addSubcommand(set)
+    return data
+  },
+  run: async function (interaction: CommandInteraction) {
+    if (interaction.options.getSubcommand() == info.name) {
```
Comment (anhnh12) [inline src/commands/config/log_slash/index.ts:22]: ===

**Review by trkhoi — APPROVED** [review]

File: src/commands/config/log_slash/info.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { composeEmbedMessage, getErrorEmbed } from "utils/discordEmbed"
+import { CommandInteraction } from "discord.js"
+import { SlashCommandSubcommandBuilder } from "@discordjs/builders"
+
+export const info = new SlashCommandSubcommandBuilder()
+  .setName("info")
+  .setDescription("Show current logging channel info")
+
+export async function logInfo(interaction: CommandInteraction) {
+  if (!interaction.guildId || !interaction.guild) {
```
Comment (anhnh12) [inline src/commands/config/log_slash/info.ts:12]:
```suggestion
  if (!interaction.guild?.id) {
```

File: src/commands/config/log_slash/set.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  .setDescription("Set logging channel to monitor guild members' activities")
+  .addStringOption((option) =>
+    option
+      .setName("channel")
+      .setDescription("the channel which you wanna log members' activities.")
+      .setRequired(true)
+  )
+
+export async function setLog(interaction: CommandInteraction) {
+  if (!interaction.guildId || !interaction.guild) {
```
Comment (anhnh12) [inline src/commands/config/log_slash/set.ts:17]: same as above

File: src/utils/discordEmbed.ts [hunk 4]
```diff
@@ -249,7 +249,7 @@ export function getSuccessEmbed(params: {
   title?: string
   description?: string
   thumbnail?: string
-  msg: Message
+  msg?: Message
```
Comment (anhnh12) [inline src/utils/discordEmbed.ts:252]: make sure this change won't break any other usages

Comment (nguyennh4522) [reply src/utils/discordEmbed.ts:252]: this function just decorates `composeEmbedMessage`, it will not break other usages

**Review by anhnh12 — APPROVED** [review]

**Pull request merged**